src/test/java/edu/hm/hafner/analysis/parser/TrivyParserTest.java

```diff
@@ -0,0 +1,31 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+class TrivyParserTest extends AbstractParserTest {
+
+    TrivyParserTest() {
+        super("trivy_result.json");
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(282);
```

Comment:
Can you please shorten the file, e.g. just 10 warnings?

Reply:
i just want to have a real world sample, but i will try to reduce...